ui/page/wallets/wallet_page.go

```diff
@@ -1204,7 +1204,7 @@ func (pg *WalletPage) HandleUserInteractions() {
 			if menu.button.Clicked() {
 				switch menu.id {
 				case SignMessagePageID:
-					pg.ParentNavigator().Display(NewSignMessagePage(pg.Load, listItem.wal))
+					pg.ParentNavigator().Display(NewSignMessagePage(pg.Load))
```

Comment:
This option should be removed since it has been migrated to the security tools page.